api/pkg/app/app.go

```diff
@@ -275,25 +288,16 @@ func initDB() (*Database, error) {
 }
 
 // initLogger returns a instance of SugaredLogger depending on the EnvMode
-func initLogger(mode EnvMode) (*zap.SugaredLogger, error) {
-
-	var log *zap.Logger
-	var err error
+func initLogger(mode EnvMode) *log.Logger {
 
 	switch mode {
 	case Production:
-		log, err = zap.NewProduction()
-
+		l, _ := zap.NewProduction()
+		return &log.Logger{SugaredLogger: l.Sugar()}
 	default:
-		config := zap.NewDevelopmentConfig()
-		config.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
-		log, err = config.Build()
-	}
-
-	if err != nil {
-		return nil, err
+		l, _ := zap.NewDevelopment()
```

Comment:
isn't the earlier code better with `CapitalColorLevelEncoder`?

Reply:
yeah changed to prev